assets/section-main-product.css

```diff
@@ -1158,6 +1158,10 @@ a.product__text {
   border-radius: calc(var(--media-radius) - var(--media-border-width));
 }
 
+.image-magnify-full-size + img {
+  opacity: 0;
```

Comment:
I think there's a transition in scope for opacity here. https://screenshot.click/02-22-zhx81-93cmp.mp4 We could override that or probably use visibility instead

Reply:
Just fixed it by adding `transition: none` to `.media>.image-magnify-hover`.

- I opted for this instead of `visibility:hidden` because `visibility` removes it from the accessibility tree.
- Applied `transition:none` to `.media>.image-magnify-hover` instead of `.image-magnify-full-size + img` because in the latter, only the `in` transition is impacted. Once the class is removed, the `out` still shows up.

Reply:
cc @sofiamatulis